templates/symfony4/files/README.md

```diff
@@ -1,21 +1,22 @@
-# Symfony 4 Flex project template for Platform.sh
+# Symfony 4 for Platform.sh
 
-This project provides a starter kit for Symfony 4 projects hosted on [Platform.sh](http://platform.sh).  There are only very minor changes from vanilla Symfony 4.
+This template provides a basic Symfony 4 skeleton.  It is configured for Production mode by default so the usual Symfony "welcome" page will not appear.  That can be adjusted in `.platform.app.yaml`.
 
-## Starting a new project
+## Services
 
-To start a new Symfony 4 project on Platform.sh, you have 2 options:
+* PHP 7.3
+* MariaDB 10.2
 
-1. Create a new project through the Platform.sh user interface and select "start new project from a template".  Then select Symfony 4 as the template. That will create a new project using this repository as a starting point.
+## Post-install
 
-2. Take an existing project, add the necessary Platform.sh files, and push it to a Platform.sh Git repository.
+1. This is a bare, empty Symfony project.  That means there will be no installation page after the site is deployed.  If you wish to switch Symfony into development mode, edit your `.platform.app.yaml` file and change the `variables.env.APP_ENV` property to "dev".  (Be sure to change it back before you go live.)
```

Comment:
Not familiar - would this same change be relevant in Symfony 3?

Reply:
No, Symfony 3 handles dev/prod mode in an entirely different way using multiple front-controller files.